x-pack/plugins/security_solution/public/management/services/exceptions_list/exceptions_list_api_client.ts

```diff
@@ -9,12 +9,18 @@ import type {
   CreateExceptionListItemSchema,
   CreateExceptionListSchema,
   ExceptionListItemSchema,
+  ExceptionListSchema,
   ExceptionListSummarySchema,
   FoundExceptionListItemSchema,
   ListId,
   UpdateExceptionListItemSchema,
 } from '@kbn/securitysolution-io-ts-list-types';
-import { EXCEPTION_LIST_ITEM_URL, EXCEPTION_LIST_URL } from '@kbn/securitysolution-list-constants';
+
+import {
+  EXCEPTION_LIST_ITEM_URL,
+  EXCEPTION_LIST_URL,
+  INTERNAL_EXCEPTIONS_LIST_ENSURE_CREATED_URL,
+} from '@kbn/securitysolution-list-constants';
 import type { HttpStart } from '@kbn/core/public';
 import { MANAGEMENT_DEFAULT_PAGE, MANAGEMENT_DEFAULT_PAGE_SIZE } from '../../common/constants';
 
```

Comment:
Is there any chance of updating this client name from `ExceptionsListApiClient` to something like `ArtifactManagementExceptionsListApiClient` - something indicating it's use for artifacts. At first glance I thought you were updating the main exceptions list client (an instance of which is also used throughout detections).

Reply:
Yeah! Let me discuss this with the rest of the team. I'm not 100% sure but this refactor could be big so I would suggest doing it in another pr soon instead of including this change here.